lib/ebae/auction.ex

```diff
@@ -0,0 +1,122 @@
+defmodule Ebae.Auction do
+  @moduledoc """
+  The Auction context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias Ebae.Repo
+  alias Ebae.{Auction.Item, Accounts, Accounts.User}
+
+  @doc """
+  Returns the list of items.
+
+  ## Examples
+
+      iex> list_items()
+      [%Item{}, ...]
+
+  """
+  def list_items do
+    Repo.all(Item)
+  end
+
+  @doc """
```

Comment:
I'm anti-doc, but 🤷‍♀

Reply:
Yeah, idk how I feel about these docs. I thought it was convention but they are quite noisy.